### PR TITLE
[feature]: Move graph component as separate submodule

### PR DESCRIPTION
**Pull request by nxanil**
@natabal @ronakmshah 

I have moved the Graph component to separate submodule. Kindly review.

### CODE REVIEW

File: src/utils/translators/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  */
 
 import demo from './demo'
-import time_convert from './time_convert'
+import time_convert from './time'
```
Comment (natabal) [inline src/utils/translators/index.js]: is there a 'time' file?

**Comment by natabal** [comment]
@nxanil Please rebase! We will start testing on this PR tomorrow

File: src/utils/translators/index.js (hunk 1 above)
Comment (natabal) [reply src/utils/translators/index.js]: @nxanil You removed the time_convert.js please rename the file only

**Comment by nxanil** [comment]
@natabal Please review the latest changes. Hope this will fix those errors.

**Pull request merged**